elixir/apps/web/lib/web/live/resources_live/edit.ex

```diff
@@ -29,242 +65,124 @@ defmodule Web.ResourcesLive.Edit do
+                placeholder="Name this Resource"
+                value={@resource.name}
                 required
               />
             </div>
             <div>
-              <.label for="name">
-                Name
+              <.label for="address">
+                Address
               </.label>
-              <input
+              <.input
+                autocomplete="off"
                 type="text"
-                name="name"
-                id="resource-name"
-                class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500"
-                value="GitLab"
+                name="address"
+                id="resource-address"
+                placeholder="Enter IP address, CIDR, or DNS name"
+                value={@resource.address}
                 required
               />
             </div>
+            <hr />
             <div class="w-full">
-              <.label for="traffic-filter">
-                Traffic restriction
-              </.label>
-              <div class="h-12 flex items-center my-4">
-                <input
+              <h3>
+                Traffic Restriction
+              </h3>
+              <div class="h-12 flex items-center mb-4">
+                <.checkbox
                   id="traffic-filter-option-1"
-                  type="radio"
                   name="traffic-filter"
                   value="none"
-                  class="w-4 h-4 border-gray-300 focus:ring-2 focus:ring-blue-300 dark:focus:ring-blue-600 dark:focus:bg-blue-600 dark:bg-gray-700 dark:border-gray-600"
+                  checked={@filter_types[:all]}
                 />
-                <label
-                  for="traffic-filter-option-1"
-                  class="block ml-4 text-sm font-medium text-gray-900 dark:text-gray-300"
-                >
+                <.label for="traffic-filter-option-1" class="ml-4 mt-2">
                   Permit all
-                </label>
+                </.label>
               </div>
               <div class="h-12 flex items-center mb-4">
-                <input
+                <.checkbox
                   id="traffic-filter-option-2"
-                  type="radio"
                   name="traffic-filter"
                   value="icmp"
-                  class="w-4 h-4 border-gray-300 focus:ring-2 focus:ring-blue-300 dark:focus:ring-blue-600 dark:focus:bg-blue-600 dark:bg-gray-700 dark:border-gray-600"
+                  checked={@filter_types[:icmp]}
                 />
-                <label
-                  for="traffic-filter-option-2"
-                  class="block ml-4 text-sm font-medium text-gray-900 dark:text-gray-300"
-                >
+                <.label for="traffic-filter-option-2" class="ml-4 mt-2">
```

Comment:
Maybe we can have some better naming here, instead of `traffic-filter-option-2` something like `filter-tcp`. It will make acceptance tests less ambiguous later.

Reply:
Sure thing.  Makes sense.

Reply:
Updated all of the labels and IDs for the traffic filters in both the new and edit pages.